src/ui/file-browser/edit/EditTextFileFormContainer.js

```diff
@@ -0,0 +1,40 @@
+import { connect } from 'react-redux';
+import { getParams } from '@entando/router';
+import { download } from 'ui/file-browser/utils/downloadFile';
+import { saveFile, downloadFile, fetchFile } from 'state/file-browser/actions';
+
+import CreateTextFileForm from 'ui/file-browser/common/CreateTextFileForm';
+
+export const mapStateToProps = state => ({
+  mode: 'edit',
+  initialValues: {
+    extension: '.txt',
```

Comment:
why does extension have an initial value of txt? what if it is a css? if the value is being retrieved by the onWillMount what is the point of the default?